include/deal.II/numerics/matrix_tools.h

```diff
@@ -68,6 +69,8 @@ namespace PETScWrappers
 #endif
 
 #ifdef DEAL_II_WITH_TRILINOS
+// TODO: move this to a stand-alone header and include from there?
+#  define TrilinosScalar double
 namespace TrilinosWrappers
```

Comment:
Add empty line.

I'm not in favor of this here, since it adds a global typedef. Why is this necessary?